src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -236,6 +236,7 @@ private boolean isContainVerifiableType(DetailAST modifiers) {
                      modifier = modifier.getNextSibling()) {
```

Comment:
This can be replaced with TokenUtils#findFirstTokenByPredicate, but it is up to you to decide.